### ISSUE TITLE
Stopwords that end in -s are counted as plurals (i.e. "was" being counted as "wa")

### ISSUE THREAD

**Issue by EpsilonAcetate** [amueller/word_cloud#570]
#### Description
Stopwords that end in -s are counted as plurals (i.e. "was" being counted as "wa", and displayed on any resulting wordclouds, if "wa" is present in the input text).

#### Steps/Code to Reproduce
In `a_file.txt`, there is one instance of the word ` wa `, and 225 instances of the word ` was ` (among lots of other text).

```python
from wordcloud import WordCloud
f = open('a_file.txt', 'r').read()
wordcloud = WordCloud().generate(f)
d = (WordCloud().process_text(f))
print(d['wa'])
wordcloud.to_file('out.png')
```
This prints 226, and in the resulting wordcloud the `wa` is very large. 

### (Suspected) Cause

This snippet in `tokenization.py` would combine the dictionary entries for `was` and `wa`:

```python
for key in list(d.keys()):
            if key.endswith('s') and not key.endswith("ss"):
                key_singular = key[:-1]
                if key_singular in d:
                    dict_plural = d[key]
                    dict_singular = d[key_singular]
                    for word, count in dict_plural.items():
                        singular = word[:-1]
                        dict_singular[singular] = (
                            dict_singular.get(singular, 0) + count)
                    merged_plurals[key] = key_singular
                    del d[key]
```

#### Versions
Darwin-18.5.0-x86_64-i386-64bit
Python 3.7.4 (default, Oct 12 2019, 18:55:28) 
[Clang 11.0.0 (clang-1100.0.33.8)]
NumPy 1.18.1
matplotlib 3.2.1
wordcoud 1.7.0

**Comment by amueller** [comment]
Thanks! Can you please provide a self-contained example to reproduce? The stopwords should be filtered out before the normalization happens.
Also, can you try the development version? It's possible the issue is fixed there.

**Comment by EpsilonAcetate** [comment]
Here's a self-contained example: 

```python
x = '''was was was was was was was was was was was was was was was 
wa 
hello hello hello hello hello hello hello hello 
goodbye 
good 
bye 
maybe 
yes 
no'''
d = WordCloud().generate(x)
d.to_file('cloud.png')
```

**Comment by amueller** [comment]
seems still to be an issue in the development version, not entirely sure why, I'll investigate.

**Comment by amueller** [comment]
Should be fixed in #571.
